docs/integrations/mlstudio_designer.md

```diff
@@ -0,0 +1,147 @@
+# Azure Machine Learning Designer Integration
+
+Connecting to the Feature Store from the Azure Machine Learning Designer requires setting up a Feature Store API key for the Designer and installing the **HSFS** on the Designer. This guide explains step by step how to connect to the Feature Store from Azure Machine Learning Designer.
+
+!!! info "Network Connectivity"
+
+    To be able to connect to the Feature Store, please ensure that the Network Security Group of your Hopsworks instance on Azure is configured to allow incoming traffic from your compute target on ports 443, 9083 and 9085. See [Network security groups](https://docs.microsoft.com/en-us/azure/virtual-network/network-security-groups-overview) for more information. If your compute target is not in the same VNet as your Hopsworks instance and the Hopsworks instance is not accessible from the internet then you will need to configure [Virtual Network Peering](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-peering).
```

Comment:
why do we not also reference our Databricks peering setup documentation?
E.g., 
For guidance, you can read [how to set up virtual peering between Databricks and Hopsworks Feature Store](https://docs.hopsworks.ai/integrations/databricks/networking/#step-1-set-up-vnet-peering-between-hopsworks-and-databricks).

Reply:
Because that one only works for Databricks. They have some special UI.

Reply:
Yes. In fact, if you connect to Hopsworks FS using the public IP, it just works without peering.

Reply:
For python clients.

Reply:
Yes but not everyone would want this to route over the Internet.